### PR TITLE
3.3.5 changes to extractor.bat

### PR DESCRIPTION
**Pull request by RainbowBridgeSoftware**
**Changes proposed:**

-  Add ability to output to a log file.
-  Some cosmetic changes.

**Tests performed:**

(Does it build, tested in-game, etc.)

I have used it to extract game files, multiple times, and have had no problems with errors
with the server or client.

### CODE REVIEW

File: contrib/extractor.bat [hunk 1]
```diff
@@ -3,57 +3,99 @@ CLS
...
 IF %M%==4 GOTO ALL
-IF %M%==5 GOTO :EOF
+IF %M%==5 GOTO ALLWITHLOGS
+IF %M%==6 GOTO :EOF
 
 :MAPS
+ECHO.
 start /b /w mapextractor.exe
+ECHO Cameras, dbc and maps extracted.
+PING 127.0.0.1 -n 5 >NUL
```
Comment (Shauren) [inline contrib/extractor.bat:39]: wtf?

Comment (RainbowBridgeSoftware) [reply contrib/extractor.bat:39]: Lets the user know the process has finished. The ping just waits five seconds before moving onto the next process.
Purely cosmetic.

File: contrib/extractor.bat [hunk 2]
```diff
@@ -3,57 +3,99 @@ CLS
...
+CLS
 GOTO MENU
 
 :VMAPS
+ECHO.
 start /b /w vmap4extractor.exe
 start /b /w vmap4assembler.exe Buildings vmaps
 rmdir Buildings /s /q
+ECHO Vmaps extracted and reassembled.
+PING 127.0.0.1 -n 5 >NUL
```
Comment (Aokromes) [inline contrib/extractor.bat:49]: what for ping?

File: contrib/extractor.bat (hunk 1 above)
Comment (Shauren) [reply contrib/extractor.bat:39]: just use timeout then
obraz

Comment (Aokromes) [reply contrib/extractor.bat:39]: then use timeout.

File: contrib/extractor.bat [hunk 3]
```diff
@@ -1,59 +1,98 @@
...
 IF %M%==4 GOTO ALL
-IF %M%==5 GOTO :EOF
+IF %M%==5 GOTO ALLWITHLOGS
+IF %M%==6 GOTO :EOF
 
 :MAPS
+ECHO.
 start /b /w mapextractor.exe
+ECHO Cameras, dbc and maps extracted.
+TIMEOUT 5 > NUL
```
Comment (Aokromes) [inline contrib/extractor.bat:36]: null

File: contrib/extractor.bat [hunk 4]
```diff
@@ -3,57 +3,99 @@ CLS
...
 :VMAPS
-start /b /w vmap4extractor.exe
-start /b /w vmap4assembler.exe Buildings vmaps
-rmdir Buildings /s /q
+ECHO.
+START /b /w vmap4extractor.exe
+START /b /w vmap4assembler.exe Buildings vmaps
+RMDIR Buildings /s /q
+ECHO Vmaps extracted and reassembled.
+TIMEOUT 5 > NUL
```
Comment (Aokromes) [inline contrib/extractor.bat:49]: there and more down

File: contrib/extractor.bat (hunk 3 above)
Comment (Shauren) [reply contrib/extractor.bat:39]: no, nul (1 L) is correct

**Pull request merged**